IGAPP-708: Noto Sans

f1sh1918 opened the pull request:
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

This PR replaces **Open-Sans** with **Noto Sans** and **Lateef** with **Noto Sans Arabic**.
Please check if the fonts gonna be displayed according to the font-config.

**Example Integreat Font Config:**

```js
const fonts: FontsType = {
  native: {
    decorativeFontBold: 'Raleway-Bold',
    decorativeFontRegular: 'Raleway-Regular',
    contentFontRegular: 'NotoSans-Regular',
    contentFontBold: 'NotoSans-Bold',
    webviewFont: 'Noto Sans, Noto Sans Arabic, Noto Sans SC, Noto Sans Georgian, sans-serif'
  },
  web: {
    decorativeFont: 'Raleway, Noto Sans Arabic, Noto Sans SC, Noto Sans Georgian, sans-serif',
    contentFont: 'Noto Sans, Noto Sans Arabic, Noto Sans SC, Noto Sans Georgian, sans-serif'
  },
  ...commonFonts
}
```

klinzo reviewed (changes requested):
![image](https://user-images.githubusercontent.com/29095487/128208480-55d3fa22-da55-42cd-a59a-a2de5bf1c0ed.png) :eyes: 

Tested on firefox and working fine. However, the native fonts are missing, therefore it is not working on iOS and probably only working on android because noto sans is the standard google/android font.

f1sh1918 commented on web/www/fonts/open-sans/LICENSE.txt:
```diff
@@ -1,202 +0,0 @@
-
```
noto sans arabic from google fonts did not provide any license. i'm gonna check this

f1sh1918 replied:
it's also SIL Open Font License. I added the file.
https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL

f1sh1918 commented on web/www/fonts/noto-sans/noto-sans.css:
```diff
@@ -0,0 +1,33 @@
+  font-family: 'Noto Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Noto Sans Regular'), local('NotoSans-Regular'),
```
yes this is the standard way to check if the user has the font files locally installed. Recommend way and also standard generation of these font css file. But the correct way is to check for 'Noto Sans' , 'NotoSans-Regular' :)
This is recommended for often used fonts

f1sh1918 commented:
> ![image](https://user-images.githubusercontent.com/29095487/128208480-55d3fa22-da55-42cd-a59a-a2de5bf1c0ed.png) 👀
> 
> Tested on firefox and working fine. However, the native fonts are missing, therefore it is not working on iOS and probably only working on android because noto sans is the standard google/android font.

Added font files and changed adjusted font-families since IOS is not using filename but font names (f.e. NotoSans)
@klinzo @SteffiStuffel 
Can you please test the fonts on ios again.
I dunno how to inspect fonts in webview. It looks fine but i can not check that properly

Taggotty reviewed (approved):
Tested on web and android

klinzo commented on native/src/utils/renderHtml.ts:
```diff
@@ -118,10 +118,16 @@ const renderHtml = (
+    @font-face {
+      font-family: 'Noto Sans Arabic';
+      font-style: normal;
+      font-weight: 400;
```
I guess:
```suggestion
      font-weight: 700;
```

f1sh1918 replied:
first one is regular 400 weight, second one is bold with 700. For me everything should be fine

klinzo commented on native/ios/Integreat/Info.plist:
```diff
@@ -45,9 +45,10 @@
-		<string>OpenSans-Bold.ttf</string>
-		<string>OpenSans-Regular.ttf</string>
+		<string>NotoSansArabic-Regular.ttf</string>
+			<string>NotoSansArabic-Bold.ttf</string>
```
```suggestion
		<string>NotoSansArabic-Bold.ttf</string>
```

klinzo commented on native/ios/Integreat.xcodeproj/project.pbxproj:
```diff
@@ -214,6 +220,11 @@
 		A330D32D22AA9F42005736D0 /* fonts */ = {
 			isa = PBXGroup;
 			children = (
+				2E0440FB26C133EE00183828 /* NotoSans.ttf */,
+				2E0440F426C127AF00183828 /* NotoSans-Bold.ttf */,
+				2E0440F526C127B000183828 /* NotoSans-Regular.ttf */,
```
Any reason why we need `NotoSans.ttf` and NotoSans-Regular.ttf`? I think we have just `NotoSans.ttf`.

f1sh1918 replied:
No i think it was not removed properly even i removed it in xcode

klinzo commented:
Tested on iOS, works fine! I think a release note would be nice for this :)
